src/main/jhipster/src/main/webapp/app/label-printing/label-printing.component.ts

```diff
@@ -333,9 +345,9 @@ export class LabelPrintingComponent implements OnInit {
 
         fileConfiguration.outputType = this.fileType.toString();
 
-        selectedFields.push($('#leftSelectedFields').sortable('toArray'));
+        selectedFields.push($('#leftSelectedFields').sortable('toArray').map((i) => Number(i)));
         if (this.fileType === FileType.PDF) {
-            selectedFields.push($('#rightSelectedFields').sortable('toArray'));
+            selectedFields.push($('#rightSelectedFields').sortable('toArray').map((i) => Number(i)));
             fileConfiguration.sizeOfLabelSheet = this.labelPrintingData.sizeOfLabelSheet;
```

Comment:
Is this because the update preset expect numbers? I would think string should be better

Reply:
It is necessary because the preset expect a list of Number.  
`public selectedFields?: number[][]`
So if I make a update into the preset and go back to select the same preset it doesn't work because the selectedFields  have string instead of numbers.